tests/views/workflow_tests.py

```diff
@@ -83,8 +84,9 @@ def test_get_workflow_history(self):
                 )
             )
 
-        self.assertEqual(response.status_code, 403)
-        self.assertIn(b"Forbidden", response.content)
+        self.assertContains(
+            response, "Permission Denied", status_code=HTTPStatus.FORBIDDEN
+        )
```

Comment:
Because this error condition was already tested, we caught an example of HTML where we should have sent JSON.

Before:

```html
\n<!doctype html>\n<html lang="en">\n<head>\n  <title>403 Forbidden</title>\n</head>\n<body>\n  <h1>403 Forbidden</h1><p></p>\n</body>\n</html>\n
```
After:
```json
{"message": "Permission Denied", "status": "false", "success": false, "title": "Request Failed"}
```